src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -580,13 +594,117 @@ ApiGlobe.prototype.setRange = function(pRange /*, bool*/ ) {
 };
 
 /**
+ * { Displaces the central point to a specific amount of pixels from its current position.
+ *  The view flies to the desired coordinate, i.e.is not teleported instantly. Note : The results can be strange in some cases, if ever possible, when e.g.the camera looks horizontally or if the displaced center would not pick the ground once displaced.}
+ *
+ * @param      {<type>}  pVector  The vector
+ */
+ApiGlobe.prototype.pan = function(pVector) {
+
+    this.scene.currentControls().pan(pVector.x,pVector.y);
+
+};
+
+/**
  * Returns the actual zoom level. The level will always be between the [getMinZoomLevel(), getMaxZoomLevel()].
- * @constructor
- * @param {Id} id - The id of a layer.
+ *
+ * @return     {<type>}  The zoom level.
+ */
+ApiGlobe.prototype.getZoomLevel = function() {
+    return this.scene.getMap().getZoomLevel();
+};
+
+
+/**
+ * Gets the current zoom level, which is an index in the logical scales predefined for the application.
+ * The higher the level, the closer to the ground.
+ * The level is always in the [getMinZoomLevel(), getMaxZoomLevel()] range.
+ *
+ * @param      {number}  zoom    The zoom
  */
+ApiGlobe.prototype.setZoomLevel = function(zoom) {
+
+    zoom = Math.max(2,zoom);
```

Comment:
Why do you use a hard-coded level?
You should compute `zoom` using `[getMinZoomLevel(), getMaxZoomLevel()]`

(and I just tested this: these 2 methods are broken :-( )